GPU/GPUTracking/TPCClusterFinder/GPUTPCNNClusterizerInternals.h

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file GPUTPCNNClusterizerInternals.h
+/// \author Christian Sonnabend
+
+#include "ML/3rdparty/GPUORTFloat16.h"
```

Comment:
I would put the #ifdefs here, and just return an empty class in case not compiled in, similar to here: https://github.com/AliceO2Group/AliceO2/blob/699feecb5261ff63bee7358c53f3505fd789c363/GPU/GPUTracking/qa/GPUQA.h#L34